### PR TITLE
Update UI

### LINKED ISSUE #39: Update Ui mock up

### PR DESCRIPTION
**Pull request by IrvingHe000**
close #39

### CODE REVIEW

**Review by fullfatwasabi — APPROVED** [review]
LGTM!

File: docs/index.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ title: Staff'd
 
 ![Ui](images/Ui.png)
 
+
 **Staff'd is a desktop application for managing your staff.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
+
```
Comment (Gabau) [inline docs/index.md:14]: Extra lines?

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM. Theres some formatting issues to address.

File: docs/index.md (hunk 1 above)
Comment (IrvingHe000) [reply docs/index.md:14]: Thank you I'll fix it :)

**Review by Gabau — APPROVED** [review]
LGTM

**Pull request merged**